### ISSUE TITLE
Vault: exiting light box mode caused 200% zooming

### ISSUE THREAD

**Issue by CyrusSUEN** [dhowe/AdNauseam#277]
[travelmama.com.hk.json.zip](https://github.com/dhowe/AdNauseam/files/329301/travelmama.com.hk.json.zip)

Import the above ad and click on it in Vault to enter light box mode, after that exit the mode by clicking anywhere else will make the Vault stays at 200%.

**Comment by dhowe** [comment]
I'm not sure this fixes the issue b/c the zooming code is a bit of a mess (see this [error], for one).
Please verify *all* vault functions

**Comment by CyrusSUEN** [comment]
This doesn't fix it. The problem is the same.

**Comment by CyrusSUEN** [comment]
Fixed: https://github.com/dhowe/AdNauseam/pull/287
Please verify

**Comment by dhowe** [comment]
ok, though I notice some 'flashing' of the image on zoom

**Comment by CyrusSUEN** [comment]
> notice some 'flashing' of the image on zoom

I've noticed it as well when implementing the zooming.

The core of our zooming implementation is to use [CSS transformation scale](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale). All the changes in vault.js is just to modify the value for the CSS attribute.
And I've logged a list of values regarding the zooming value. No outlier values can be observed in the log (they were in either in ascending or descending order).
